### ISSUE TITLE
None handling

### ISSUE THREAD

**Issue by WolfgangFahl** [tholzheim/orapi#34]
https://confident.dbis.rwth-aachen.de/or/index.php?title=ACSAC
has quite a few invalid "None" entries at different positions even in a URL. This behavior should be avoided.

**Comment by tholzheim** [comment]
Yeah, the None values were not filtered out before handing over the entity to the Wikifile api
